deployment/puppet/Puppetfile.lock

```diff
@@ -43,7 +42,7 @@ FORGE
       puppetlabs-stdlib (< 6.0.0, >= 4.13.1)
     puppetlabs-powershell (2.3.0)
     puppetlabs-reboot (2.1.2)
-    puppetlabs-stdlib (4.25.1)
+    puppetlabs-stdlib (5.2.0)
```

Comment:
Can we push it to 6.0.0 (the latest version) by any chance?
https://forge.puppet.com/puppetlabs/stdlib

Reply:
Since it's a dependency, isn't it automatically resolved?